### ISSUE TITLE
Animations not loading in prod build

### ISSUE THREAD

**Issue by caiomcoliveira** [ngx-lottie/ngx-lottie#175]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Running lottie locally shows my animation, but building in prod throws and error.
```
polyfills-RX4V3J3S.js:1 Uncaught TypeError: i.useWebWorker is not a function
    at Object.next (main-IBHDEVCE.js:8:112084)
    at i.subscribe.a (main-IBHDEVCE.js:3:23814)
    at Js._next (main-IBHDEVCE.js:3:5425)
    at Js.next (main-IBHDEVCE.js:3:2519)
    at t._subscribe (main-IBHDEVCE.js:3:16511)
    at t._trySubscribe (main-IBHDEVCE.js:3:4415)
    at main-IBHDEVCE.js:3:4360
    at Dn (main-IBHDEVCE.js:3:2212)
    at t.subscribe (main-IBHDEVCE.js:3:4276)
    at main-IBHDEVCE.js:3:23757
```
![image](https://github.com/ngx-lottie/ngx-lottie/assets/14111942/413decbf-2079-4d84-828f-2d471b07c5a3)

## Expected behavior

Production build to work normally

## Minimal reproduction of the problem with instructions

* Put the json file into the assets folder
* Load the animation normally in dev (Using standalone components)
* Build the application and deploy it

--- Detailed Reproduction
```
ng new lottie-bug
npm i ngx-lottie
```

Add the provider under
app.config.ts
```
  provideLottieOptions({
      player: () => import(/* webpackChunkName: 'lottie-web' */ 'lottie-web'),
    }),

```

import lottie under the app.component.ts
```
import { AnimationOptions, LottieComponent } from 'ngx-lottie';
LottieComponent
```

```
Add a animations to assets/animation.json
```

Put this on app.component.html
```
<ng-lottie height="200px" [options]="{
  path: 'assets/animation.json',
}"/>
```

Now build the app, and serve it using a http client
Go to the dist folder of the built app
``` 
npx http-server
```

## What is the motivation / use case for changing the behavior?
It doesn't work in prod built applications using angular 17

## Environment

<pre><code>
Libs:
- @angular/core version: 17
- ngx-lottie version: 10.0.0

Browser:
- [X] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by arturovt** [comment]
Please fill in the issue template and provide a minimal reproducible example.

**Comment by caiomcoliveira** [comment]
> Please fill in the issue template and provide a minimal reproducible example.
I couldnt exactly reproduce it on stack blitz
But I included a step by step reproduction from a brand new project

**Comment by arturovt** [comment]
This is not reproducible on a brand new project.

**Comment by Bafetimbi** [comment]
Same issue here.

**Comment by arturovt** [comment]
Closing until the reproducible example is provided, feel free to re-open once you have the runnable example.

**Comment by Bafetimbi** [comment]
- `npm init @angular myApp` with SSR enabled (angular version 17.1.2)
- `npm i lottie-web ngx-lottie`
- `app.component.ts`:

```
import { Component } from '@angular/core';
import { RouterOutlet } from '@angular/router';
import { AnimationOptions, LottieComponent } from "ngx-lottie";

@Component({
  selector: 'app-root',
  standalone: true,
  imports: [RouterOutlet, LottieComponent],
  template: '<ng-lottie [options]="options"></ng-lottie>',
  styleUrl: './app.component.scss'
})
export class AppComponent {
  title = 'myApp';

  options: AnimationOptions = {
    path: '/assets/animation.json',
  };
}
```

- `app.config.ts`:

```
import { ApplicationConfig } from '@angular/core';
import { provideRouter } from '@angular/router';

import { routes } from './app.routes';
import { provideClientHydration } from '@angular/platform-browser';
import { provideLottieOptions } from "ngx-lottie";

export const appConfig: ApplicationConfig = {
  providers: [
    provideRouter(routes),
    provideClientHydration(),
    provideLottieOptions({
    player: () => import(/* webpackChunkName: 'lottie-web' */ 'lottie-web'),
    })
  ]
};
```

- `npm run build`
- `npm run serve:ssr:myApp`

And there it is.

**Comment by arturovt** [comment]
Published `ngx-lottie@11.0.0`.

**Comment by Bafetimbi** [comment]
Works like a charm now, thank you @arturovt ! 😊
